dist/assets/meta-data.hbs

```diff
@@ -0,0 +1,37 @@
+<!-- Favicon -->
+<link rel="icon" type="image/x-icon" href="{{favicon16}}" sizes="16x16">
```

Comment:
can this direct reference the files and have {{favicon16}} etc be the 'cdn/localpath' like in head.hbs of qgds-bootstrap5.

Reply:
i.e.
 href="{{#if cdn }}{{#ifCond cdn '==' 'PROD'}}https://static.qgov.net.au/qgds-tokens/###MAJOR_VERSION###/###MAJOR_VERSION###.x.x-latest{{else}}{{#ifCond cdn '==' 'STAGING'}}https://staging-static.qgov.net.au/qgds-tokens/###MAJOR_VERSION###/###MAJOR_VERSION###.x.x-latest{{else}}{{#ifCond cdn '==' 'BETA'}}https://beta-static.qgov.net.au/qgds-tokens/###MAJOR_VERSION###/###MAJOR_VERSION###.x.x-latest{{else}}{{#ifCond cdn '==' 'TEST'}}https://test-static.qgov.net.au/qgds-tokens/###MAJOR_VERSION###/###MAJOR_VERSION###.x.x-latest{{else}}{{#ifCond cdn '==' 'DEV'}}https://dev-static.qgov.net.au/qgds-tokens/###MAJOR_VERSION###/###MAJOR_VERSION###.x.x-latest{{else}}{{cdn}}{{/ifCond}}{{/ifCond}}{{/ifCond}}{{/ifCond}}{{/ifCond}}{{else}}.{{/if}}/favicons/128x128-favicon.png">

if cdn is set to a path it uses relative, if set to a keyword, it uses the cdn, and when missing, just . so it still functions.

for testing, we will need to get storybook stood up in this repo or at least a unit/selenium test to verify the dom works and files do exist, etc.